tests/v2/actions/secrets/secrets.go

```diff
@@ -0,0 +1,52 @@
+package secrets
+
+import (
+	"fmt"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/extensions/secrets"
+	namegen "github.com/rancher/shepherd/pkg/namegenerator"
+	"github.com/rancher/shepherd/pkg/wrangler"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// CreateSecret is a helper to create a secret
+func CreateSecret(client *rancher.Client, clusterID, namespaceName string) (*corev1.Secret, error) {
```

Comment:
IMO we can leave this as a action, given that it's a generic feature within Rancher and has the potential to be utilized by other tests as well.

Reply:
ok, thinking more about this since this is a generic function and can be used in other tests F.E: if we try to create a global role with namespace rule and this resource in rule, it can be useful to verify few things. we can have it here. Same goes to configmap also and I will move that to actions.